packages/openapi-ts/src/compiler/classes.ts

```diff
@@ -156,47 +156,65 @@ type ClassDecorator = {
  * @returns ts.ClassDeclaration
  */
 export const createClassDeclaration = ({
-    decorator = undefined,
-    members = [],
-    name,
+  decorator = undefined,
+  members = [],
+  name
 }: {
-    decorator?: ClassDecorator;
-    members?: ts.ClassElement[];
-    name: string;
+  decorator?: ClassDecorator
+  members?: ts.ClassElement[]
+  name: string
 }) => {
-    const modifiers: ts.ModifierLike[] = [ts.factory.createModifier(ts.SyntaxKind.ExportKeyword)];
-    if (decorator) {
-        modifiers.unshift(
-            ts.factory.createDecorator(
-                ts.factory.createCallExpression(
-                    ts.factory.createIdentifier(decorator.name),
-                    undefined,
-                    decorator.args.map(arg => toExpression({ value: arg })).filter(isType<ts.Expression>)
-                )
-            )
-        );
-    }
-    // Add newline between each class member.
-    const m: ts.ClassElement[] = [];
-    members.forEach(member => {
-        m.push(member);
-        // @ts-ignore
-        m.push(ts.factory.createIdentifier('\n'));
-    });
-    return ts.factory.createClassDeclaration(modifiers, ts.factory.createIdentifier(name), [], [], m);
-};
+  const modifiers: ts.ModifierLike[] = [
+    ts.factory.createModifier(ts.SyntaxKind.ExportKeyword)
+  ]
+  if (decorator) {
+    modifiers.unshift(
+      ts.factory.createDecorator(
+        ts.factory.createCallExpression(
+          ts.factory.createIdentifier(decorator.name),
+          undefined,
+          decorator.args
+            .map(arg => toExpression({ value: arg }))
+            .filter(isType<ts.Expression>)
+        )
+      )
+    )
+  }
+  // Add newline between each class member.
+  const m: ts.ClassElement[] = []
+  members.forEach(member => {
```

Comment:
Also prefer brackets around arguments, but not super strongly

Reply:
All changes are based on the prettier config. We can tweak the config after this is merged maybe? Just since it's a big diff?

Reply:
It seems to be changing some of the existing formatting, might as well do it in this pull request

Reply:
I did it based on the config provided in the issue. What wouldd you prefer changes in that?